frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -68,6 +91,26 @@ async function deleteUserRoleAssignment(
         );
     }
 }
+
+const deleteAppAdmin = async ( admin: FamAppAdminGet) => {
+    try {
+        applicationAdmins.value = await deleteAndRefreshApplicationAdmin(
+            admin.application_admin_id,
+            admin.application_id
```

Comment:
and I think we shouldn't pass "admin.application_id". So the deleteAndRefreshApplicationAdmin method will call fetchApplicationAdmin with application_id to get the admins again after deletion, and this application id is what we selected from dropdown, which is the id of "selectedApplication". However, admin.application_id is the application this user is admin of, so it could be the id for fom, forest client or spar, and we can't use it. 

So we might don't need to pass the application id, inside the deleteAndRefreshApplicationAdmin, we can just call fetchApplicationAdmin(FAM_APPLICATION_ID), where FAM_APPLICATION_ID should be a new constant variable we created in `frontend/src/store/Constants.ts`

Reply:
ok, I'll make that refactor